src/vscode-dts/vscode.d.ts

```diff
@@ -10110,9 +10110,11 @@ declare module 'vscode' {
 
 		/**
 		 * Allows iteration through the data transfer items.
+		 *
 		 * @param callbackfn Callback for iteration through the data transfer items.
+		 * @param thisArg The `this` context used when invoking the handler function.
 		 */
-		forEach(callbackfn: (value: DataTransferItem, key: string) => void): void;
+		forEach(callbackfn: (value: DataTransferItem, key: string) => void, thisArg?: unknown): void;
```

Comment:
`unknown` or `any`?

Reply:
hm current apis use both. Let's switch them all to `any` since that's what TS uses for the sodlib `d.ts`